CMakeModules/eosio-config.cmake.in

```diff
@@ -1,8 +1,10 @@
-if(EOSIO_ROOT STREQUAL "" OR NOT EOSIO_ROOT)
-   set(EOSIO_ROOT "@EOS_ROOT_DIR@")
+if(NOT EOSIO_ROOT)
+   list(APPEND CMAKE_MODULE_PATH @EOS_ROOT_DIR@/cmake/eosio)
+else()
+   # legacy behavior for anyone configuring with -DEOSIO_ROOT=/foo or such
+   list(APPEND CMAKE_MODULE_PATH ${EOSIO_ROOT}/lib/cmake/eosio)
+   list(APPEND CMAKE_MODULE_PATH ${EOSIO_ROOT}/lib64/cmake/eosio)
 endif()
-list(APPEND CMAKE_MODULE_PATH ${EOSIO_ROOT}/lib/cmake/eosio)
-list(APPEND CMAKE_MODULE_PATH ${EOSIO_ROOT}/lib64/cmake/eosio)
 include(EosioTester)
```

Comment:
this section above could use some :eyes:

Reply:
If `EOSIO_ROOT` is not defined, would it be likely `EOS_ROOT_DIR` not defined either? It is safer to check if `EOS_ROOT_DIR` is defined before use.